### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.NullPointerException

### ISSUE THREAD

**Issue by konebra** [microsoft/azure-tools-for-java#2893]
IntelliJ build version: 2018.3.5 IC-183.5912.21
OS: Mac OS X
JDK: JetBrains s.r.o 1.8.0_152-release
Plugin version: 3.19.0-2018.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,22,1280x755,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.NullPointerException
	at com.intellij.openapi.wm.impl.InternalDecorator.setTitleActions(InternalDecorator.java:316)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.setTitleActions(ToolWindowImpl.java:309)
	at com.microsoft.intellij.components.ServerExplorerToolWindowFactory.lambda$addToolbarItems$3(ServerExplorerToolWindowFactory.java:374)
	at com.microsoft.azuretools.authmanage.AuthMethodManager.notifySignInEventListener(AuthMethodManager.java:97)
	at com.microsoft.azuretools.ijidea.actions.AzureSignInAction.doSignIn(AzureSignInAction.java:133)
	at com.microsoft.azuretools.ijidea.actions.AzureSignInAction.onAzureSignIn(AzureSignInAction.java:117)
	at com.microsoft.intellij.serviceexplorer.azure.SignInOutAction$1.actionPerformed(SignInOutAction.java:46)
	at com.microsoft.tooling.msservices.serviceexplorer.NodeActionListener.actionPerformedAsync(NodeActionListener.java:72)
	at com.microsoft.tooling.msservices.serviceexplorer.No

**Comment by wezhang** [comment]
@Flanker32 , could you help follow it!

**Comment by Flanker32** [comment]
@konebra  Thanks for your report, could you please share me which method did you use to login? And is there any method could reproduce this issue?

**Comment by Flanker32** [comment]
We didn't remove the tool window listener when close a project, fixed in #3431
